### PR TITLE
IGAPP-932: Links opened in browser

### PR DESCRIPTION
**Pull request by klinzo**
This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **IGAPP**.

### CODE REVIEW

**Comment by klinzo** [comment]
Not really nice but a working workaround. There were two problem: 
* Links were always opened in the browser with the upgrade to v11.0 (see https://github.com/react-native-webview/react-native-webview/releases/tag/v11.0.0,done in #542), not sure how we could miss that. I am fairly sure I tested this quite much :crying_cat_face: Setting `setSupportMultipleWindows={false}` fixes this.
* The issue mentioned [here](https://issues.tuerantuer.org/browse/IGAPP-462) resurfaced after fixing the first problem, i.e. the pressed url is loaded in the webview even though we return false in `onShouldStartLoadRequest`. This seems to happen if it takes too long returning false. Therefore I implemented the workaround of only setting the pressed url to the state and opening it later in `useEffect`.

**Comment by klinzo** [comment]
Please test this thoroughly! Only happens on android.

File: native/src/components/RemoteContent.tsx [hunk 1]
```diff
@@ -29,19 +29,30 @@ type PropType = {
...
+  const [pressedUrl, setPressedUrl] = useState<string | null>(null)
   // https://github.com/react-native-webview/react-native-webview/issues/1069#issuecomment-651699461
   const defaultWebviewHeight = 1
   const [webViewHeight, setWebViewHeight] = useState<number>(defaultWebviewHeight)
-  const [error, setError] = useState<string | null>(null)
   const { width } = useWindowDimensions()
   const webViewWidth = width - 2 * dimensions.page.horizontalMargin
+  const theme = useTheme()
+
+  useEffect(() => {
```
[the remark this answers is not in the document]
Comment (klinzo) [reply native/src/components/RemoteContent.tsx:47]: Android or iOS?

[the remark this answers is not in the document]
Comment (f1sh1918) [reply native/src/components/RemoteContent.tsx:47]: gonna check it. atm can not build for ios on this branch :/

**Review by f1sh1918 — APPROVED** [review]
tested on android

File: native/src/components/RemoteContent.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (klinzo) [reply native/src/components/RemoteContent.tsx:47]: To test this:
* Click on a link
* Navigate back once
* Webview has loaded the link as well (even though it should not)

Android only

**Review by Taggotty — APPROVED** [review]
Thoroughly tested on Android device

**Pull request merged**